tests/test-policies/src/test/java/org/mule/test/policy/PolicyTestCase.java

```diff
@@ -16,30 +16,43 @@
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.not;
 import static org.hamcrest.Matchers.nullValue;
-
+import static org.mule.tck.probe.PollingProber.check;
 import org.mule.functional.junit4.MuleArtifactFunctionalTestCase;
 import org.mule.runtime.api.config.custom.ServiceConfigurator;
+import org.mule.runtime.api.exception.MuleException;
+import org.mule.runtime.api.lifecycle.Disposable;
 import org.mule.runtime.api.lifecycle.Startable;
 import org.mule.runtime.api.scheduler.SchedulerService;
 import org.mule.runtime.core.api.MuleContext;
 import org.mule.runtime.core.api.config.ConfigurationBuilder;
 import org.mule.runtime.core.api.config.ConfigurationException;
+import org.mule.runtime.core.api.construct.Flow;
+import org.mule.runtime.core.api.event.CoreEvent;
 import org.mule.runtime.core.api.policy.Policy;
 import org.mule.runtime.core.api.policy.PolicyInstance;
 import org.mule.runtime.core.api.policy.PolicyProvider;
+import org.mule.runtime.core.api.processor.Processor;
 import org.mule.runtime.policy.api.PolicyPointcutParameters;
+import org.mule.tck.junit4.rule.DynamicPort;
 
 import java.util.List;
 import java.util.Optional;
+import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicReference;
 
 import javax.inject.Inject;
 
+import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
 
 public class PolicyTestCase extends MuleArtifactFunctionalTestCase {
 
   private static final String POLICY_ID = "policyId";
+  private static final AtomicBoolean processorWasDisposed = new AtomicBoolean(false);
+
+  @Rule
+  public DynamicPort proxyPort = new DynamicPort("listenerPort");
```

Comment:
why this?

Reply:
Was there because I copied part of an existing test. Just deleted it.